README.md

```diff
@@ -447,6 +447,36 @@ It's important that `x-if` is used on a `<template></template>` tag because Alpi
 
 > Note: `x-if` must have a single element root inside the `<template></template>` tag.
 
+> Note: When using `template` in a `svg` tag, you need to add a polyfill that should be run before Alpine.js is initialized:
+>
+> ```html
+> <script>
+> (function() {
+>     var templates = document.querySelectorAll('svg template');
+>     var el, template, attribs, attrib, count, child, content;
+>     for (var i = 0; i < templates.length; i++) {
+>         el = templates[i];
+>         template = el.ownerDocument.createElement('template');
+>         el.parentNode.insertBefore(template, el);
+>         attribs = el.attributes;
+>         count = attribs.length;
+>         while (count-- > 0) {
+>             attrib = attribs[count];
+>             template.setAttribute(attrib.name, attrib.value);
+>             el.removeAttribute(attrib.name);
+>         }
+>         el.parentNode.removeChild(el);
+>         content = template.content;
+>         while ((child = el.firstChild)) {
+>             content.appendChild(child);
+>         }
+>     }
+> })();
```

Comment:
I don't think we should include the polyfill inline, link to the comment that has the polyfill code instead maybe? https://github.com/alpinejs/alpine/issues/637#issuecomment-654856538

Reply:
I think it's better to include it in case the comment that includes it gets removed (just in case). I'll let Caleb decide on that :slightly_smiling_face:

Reply:
Hmmm yeah, I'm just thinking that adding code inline is quite noisy (especially for people who don't use Alpine with svg), potentially you could create a Gist and link to it if you're worried about things dissappearing

Reply:
Thanks for the documentation note. I'd love to just include the link and not the entire snippet like Hugo said.

Thanks!

Reply:
nice one @Calinou